input/pagecontent/use-cases.md

```diff
@@ -1,7 +1,10 @@
 ## AU eRequesting R1 use cases
-This page contains the use cases for AU eRequesting for the first release of the TDG Implementation Guide (R1). This is a working page and will reflect the current state of agreement by the TDG as well as the current draft of the use case material.
-
-{% include use-case-feedback.md -%}
+This page contains the use cases for AU eRequesting for the first release of the TDG Implementation Guide (R1).
+Column attribute descriptions are as follows:
```

Comment:
1. In ID, remove word "Represents"
2. In Description remove word "detailed" and "represents" Just say "Narrative of ..."
3. Change to just be "Data Group(s)" instead of "AUeReqDI/AUCDI Data Group(s)"
4. In text of "Data Group(s): Remove "Represents the"

Reply:
Done

Reply:
Confirmed 1-4. Further improvements can be done post ballot snapshot.